core/vmpu/src/mpu_armv7m/vmpu_armv7m_mpu.c

```diff
@@ -138,7 +138,7 @@ uint8_t vmpu_region_bits(uint32_t size)
 {
     uint8_t bits;
 
-    bits = vmpu_bits(size) - 1;
+    bits = (0 == size) ? 0: (vmpu_bits(size) - 1);
```

Comment:
Do we want vmpu_region_bits to return 0 for size 0? It's getting rounded up from 0 to 1, as currently written. Probably we should just consider size 0 as a special case, perhaps asserting that size is not 0.

Reply:
I'm in favor of asserting, there is no reason size should be 0.